### PR TITLE
🚧Refactor WindowSizeEventArgs ctor param data types

### PR DESCRIPTION
**Pull request by SyedMSawaid**
<h1 style="font-weight:bold" align="center">Preview Feature Pull Request</h1>
<h2 style="font-weight:bold" align="center">📋Review Checklist📋</h2>

- [x] The **_head(source)_** branch for this pull request is a **_preview feature_** branch, with the correct naming syntax, in the following example:
  - A **_head(source)_** branch is the branch being merged into a target branch.
    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/feature/<issue-num>-<description>
      Example: preview/feature/123-my-preview-feature
      ```
    </details>

- [x] The **_head(source)_** branch for this pull request is created from a **_preview release_** branch, with the correct naming syntax, in the following example:
  <details closed><summary>Example</summary>

    ``` xml
    Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
    Example: preview/v1.2.3-preview.4
    ```
  </details>

- [x] The **_base(target)_** branch for this pull request is a **_preview release_** branch, with the correct naming syntax, in the following example:
  - A **_base(target)_** branch is the branch that the **_head(source)_** branch is merging into.

    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
      Example: preview/v1.2.3-preview.4
      ```
    </details>

💡For more information on branching, refer to the project's [branching documentation](../../Documentation/Branching.md).

- [x] Associated issue exists and is linked to this pull request.
  - One issue per pull request.

- [x] Pull request title matches the title of the associated issue.

- [x] A `preview` label has been added to the pull request.

- [x] The labels attached to this PR match the labels attached to the associated issue.

- [x] Project assigned matches the project of the associated issue.

- [x] Milestone assigned matches the milestone of the associated issue.

- [x] My code follows the coding style of this project.
  - The style is enforced by the **_.editorconfig_** files in the project and displayed as warnings.  If there is an edge case, reach out, and let's discuss it.

- [x] I have manually tested my code changes to the best of my ability.

- [x] All tests passed locally.
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
  - Tests might pass on the developer's machine but not necessarily on the code reviewer's machine.
  - Status checks run for this pull request when merging into the **_preview/v\*.\*.\*-preview.\*_** branch.  These status checks run every time a change is pushed to the pull request.  These checks validate version syntax, tagging, builds, unit tests, and more.

### CODE REVIEW

**Comment by SyedMSawaid** [comment]
@CalvinWilkinson I also added some directories to `.gitignore`. I hope that doesn't cause any issues.

File: .gitignore [hunk 1]
```diff
@@ -351,3 +351,7 @@ MigrationBackup/
 
 # Ignore secrets file used for CICD build system
 local-secrets.json
+
+# Ignore Visual Studio Code files
+.vscode/launch.json
+.vscode/tasks.json
```
Comment (CalvinWilkinson) [inline .gitignore:357]: Please remove these changes.  I do plan on eventually setting up some VSCode for contributors who would like to use VSCode.

**Review by CalvinWilkinson — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (CalvinWilkinson) [inline .gitignore:357]: Please remove these changes.  I do plan on eventually setting up some VSCode for contributors who would like to use VSCode.

```suggestion
```

**Review by CalvinWilkinson — CHANGES_REQUESTED** [review]

**Comment by CalvinWilkinson** [comment]
@SyedMSawaid 
Just a quick heads up.  I noticed that you still do not have a preview branch with the name _**preview/v1.0.0-preview.12**_ in your fork.  

It will not be a problem with this issue so you don't need to do anything.  Next time you have to make sure you pull in the preview branch from the original repo into your fork before creating your branch.  If you don't then you might be creating your branch from a branch that would have changes that you do not want to be inherited into your branch.

If you need assistance on how to do that, I can help you out.
I have finished the contribution changes as well so it has everything you need in there for reference as well.  

The `CONTRIBUTING.md` file on the `preview/v1.0.0-preview.12` branch has all of the info you need.

File: .gitignore (hunk 1 above)
Comment (SyedMSawaid) [reply .gitignore]: I removed them.

File: Testing/VelaptorTests/OpenGL/WindowSizeEventArgsTests.cs [hunk 2]
```diff
@@ -18,15 +18,15 @@ public class WindowSizeEventArgsTests
...
 
             // Act
             var actualWidth = eventArgs.Width;
             var actualHeight = eventArgs.Height;
 
             // Assert
-            actualWidth.Should().Be(123);
-            actualHeight.Should().Be(456);
+            Assert.Equal(123u, actualWidth);
+            Assert.Equal(456u, actualHeight);
```
Comment (CalvinWilkinson) [inline Testing/VelaptorTests/OpenGL/WindowSizeEventArgsTests.cs:29]: Please revert this code back to using **FluentAssertions**

**Review by CalvinWilkinson — CHANGES_REQUESTED** [review]

File: Testing/VelaptorTests/OpenGL/WindowSizeEventArgsTests.cs (hunk 2 above)
Comment (SyedMSawaid) [reply Testing/VelaptorTests/OpenGL/WindowSizeEventArgsTests.cs:29]: I will fix it tomorrow morning.

**Comment by SyedMSawaid** [comment]
PR seems good to me.

**Review by CalvinWilkinson — APPROVED** [review]

**Pull request merged**